packages/components/modal/README.md

```diff
@@ -33,12 +33,14 @@ table {
 
 ## ➤ Properties
 
-| Property | Attribute    | Type    | Default | Description                                                    |
-| -------- | ------------ | ------- | ------- | -------------------------------------------------------------- |
-| dir      | data-dir     | string  | ''      | Direction 'rtl' or 'ltr'                                       |
-| size     | data-size    | string  | 'large' | Size of the modal. Available two variants: 'large' or 'medium' |
-| title    | data-title   | string  | ''      | Modal header text                                              |
-| visible  | data-visible | boolean | false   | Show/hide the modal                                            |
+| Property   | Attribute    | Type    | Default | Description                                                    |
+| ---------- | ------------ | ------- | ------- | -------------------------------------------------------------- |
+| dir        | data-dir     | string  | ''      | Direction 'rtl' or 'ltr'                                       |
+| size       | data-size    | string  | 'large' | Size of the modal. Available two variants: 'large' or 'medium' |
+| title      | data-title   | string  | ''      | Modal header text                                              |
+| visible    | data-visible | boolean | false   | Show/hide the modal                                            |
+| hideHeader | hide-header  | boolean | false   | Show/hide thetitle of the modal                                |
```

Comment:
Deliberately didn't add this to readme, to not encourage people for using it. Not sure if it's actually a good idea or not! You think we should add it?

Reply:
I'm not sure that we should hide any public API that components have, because:
- I doubt that developers will use it without a real need (e.g. they usually have a design from UI/UX team),
- Most curious developers will find it anyway, because it is already in use in some apps.
I think we can create a warning mark for properties that we don't encourage to use.

Reply:
You are right but not all designs follow the principle to consider consistency, that much, and in this way we have the chance to at least stop some misuses.
Let's keep it then 👍